.github/workflows/filter_testsuite.yml

```diff
@@ -0,0 +1,105 @@
+name: Filter Testsuite
+
+on:
+  issue_comment:
+      types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/github-script@v3
+        id: check
+        with:
+         result-encoding: string
+         script: |
+           const userName = context.payload.comment.user.login
+           if(userName == 'maxGimeno') {
+             const body = context.payload.comment.body
+             if(body.includes("/testme")) {             
+               return 'OK'
+             }
+           }
+           return 'stop'
+      - uses: actions/github-script@v3
+        if: steps.check.result != 'stop'
+        id: get_label
+        with:
+         result-encoding: string
+         script: |
+           //get branch name and username
+           const pr_url = context.payload.issue.pull_request.url
+           const pr_content = await github.request(pr_url)
+           const label = pr_content.data.head.label
+           console.log(label)
+           return label
+      - uses: actions/github-script@v3
+        if: steps.check.result != 'stop'
+        id: get_base
+        with:
+         result-encoding: string
+         script: |
+           //get branch name and username   
+           const pr_url = context.payload.issue.pull_request.url
+           const pr_content = await github.request(pr_url)
+           const base = pr_content.data.base.ref
+           console.log(base)
+           return base
+      - uses: actions/github-script@v3
+        if: steps.get_round.outputs.result != 'stop'
+        id: get_pr_number
+        with:
+         result-encoding: string
+         script: |
+           //get pullrequest url
+           const pr_number = context.payload.issue.number
+           return pr_number
```

Comment:
Can be removed. `issue.number` is directly a value in the payload. It can be accessed using the syntax `${{...}}}`.